projects/opendal/build.sh

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# create env file
+echo -e "OPENDAL_MEMORY_TEST=on\nOPENDAL_FS_TEST=on\nOPENDAL_FS_ROOT=/tmp" > $SRC/opendal/.env
```

Comment:
We don't need to to create .env file here. Instea, we just need to add `ENV OPENDAL_MEMORY_TEST=on` in Docker.

cc @dqhl76

Reply:
same with https://github.com/google/oss-fuzz/pull/10794#discussion_r1286508948